src/dstack/_internal/utils/gpu.py

```diff
@@ -20,8 +20,10 @@ def convert_gpu_name(name: str) -> str:
         return name.replace(" ", "")
 
     name = name.replace(" Ti", "Ti")
+    name = name.replace(" NVL", "NVL")
+    name = name.replace(" Ada Generation", "Ada")
     name = name.replace("RTX ", "RTX")
-    m = re.search(r"([A|H|L|P|T|V]\d+[Ti]?)", name)
+    m = re.search(r"([A|H|L|P|T|V]\d+\w*)", name)
```

Comment:
Not directly related to this PR, but `[A|H|L|P|T|V]` should be `[AHLPTV]`, `|` has no special meaning inside sets, it's just a literal bar.